blog/2024/2024-03-15-bestpractices-emailsupport.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: Best Practices Series - Email Support
+title: Best Practices Series - Email Support
+authors:
```

Comment:
We only use authors when we have several authors, please use:
`author: Vikas Gupta`